execution/leveldb.go

```diff
@@ -0,0 +1,125 @@
+package execution
+
+import (
+	"crypto/sha1"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/cnf/structhash"
+	"github.com/mesg-foundation/core/service"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+// LevelDB is a concrete implementation of the DB interface
+type LevelDB struct {
```

Comment:
Change name as it's not levelDB struct, rather ExecutionDB

Reply:
I think we should keep the `levelDB`, I can (as the service db package) append the executionDB to have `LevelDBExecutionDB` but we need to keep the kind of implementation in the name.